src/api.ts

```diff
@@ -3,6 +3,29 @@ import { Person } from './types/Person';
 // eslint-disable-next-line max-len
 const API_URL = 'https://mate-academy.github.io/react_people-table/api/people.json';
 
+export const NOT_SET_VALUE = '-';
+
+export enum ColumnNames {
+  Name = 'Name',
+  Sex = 'Sex',
+  Born = 'Born',
+  Died = 'Died',
+}
+
+export enum PersonSex {
+  All = '',
+  Male = 'm',
+  Female = 'f',
+}
+
+export type FilterType = {
+  query: string | '';
+  centuries: string[];
+  sex: string | '';
+};
```

Comment:
```suggestion
export type FilterType = {
  query: string;
  centuries: string[];
  sex: string;
};
```
An empty string is still a string